server/app.js

```diff
@@ -16,11 +15,26 @@ app.get('/site/:id', (req, res) => {
   var campid = req.params.id;
   Camp.find({ id: campid }).exec((err, campsite) => {
     if (err) {
-      res.status(404).send({ 'database-error': err });
+      res.status(404).send(err);
```

Comment:
In general it's bad practice to send the err object back to the client.  The err object will contain all sorts of information about the server that might pose a security risk if publicly available. Instead, you want to send back either a short informative message, or something non-revealing that the client can easily interpret. Often it's fine to just let the status code do the heavy lifting.

Reply:
Thanks for your reviewing and comment. I usually do sending error message back while I am testing my own server. It's good to know that it's a bad practice. I'll change this to something non-revealing.